### PR TITLE
update core.js

### PR DESCRIPTION
**Pull request by yishuzhang0302**
change the color and width for expression, so that user may more clear about what to click/drag? Feel free to change it back if you think it looks a little weird now. Thanks a lot!

### CODE REVIEW

**Comment by lidavidm** [comment]
@yishuzhang0302 do you want to post a picture on Slack for Myers/Erik/Francois to look at?

**Comment by yishuzhang0302** [comment]
Sure! I have just posted it in #progression

File: src/stage.js [hunk 1]
```diff
@@ -86,12 +86,12 @@ class TouchRecord {
             // Click on object to reduce
             let selectedNode = this.topNode;
 
-            if (this.targetNode) {
+            /*if (this.targetNode) {
```
Comment (lidavidm) [inline src/stage.js]: Heads up, you can just go ahead and delete the code - we can always restore it from Git if needed (just use a descriptive commit message so it's easy to find)

**Comment by lidavidm** [comment]
Going to merge this for now, we can come back and tweak the border later.

**Pull request merged**